error.go

```diff
@@ -12,16 +12,18 @@ import (
 // This is used at sites in the code where it is unknown whether the outcome
 // should result in a runtime exit or in continued execution with a
 // Missing value embedded in the Zed results.
-var ErrMissing = errors.New(missing)
+var ErrMissing = errors.New("missing")
 
-const missing = "missing"
+var missing = []byte("missing")
+var quiet = []byte("quiet")
 
 // Missing is value that represents the error condition that a field
 // referenced was not present.  The Missing value can be propagated through
 // functions and expressions and each operator must clearly defined its
 // semantics with respect to the Missing value.  For example, "true AND MISSING"
 // is MISSING.
-var Missing = NewError(ErrMissing)
+var Missing = &Value{TypeError, missing}
+var Quiet = &Value{TypeError, quiet}
```

Comment:
NIt: There's no benefit to having `var missing = ...` and `var quiet = ...` up above.
```suggestion
var Missing = &Value{TypeError, []byte("missing")}
var Quiet = &Value{TypeError, []byte("quiet")}
```

Reply:
It's for IsMissing and IsQuiet with one place to define the message.